packages/svelte2tsx/svelte-shims.d.ts

```diff
@@ -49,8 +58,23 @@ declare function __sveltets_restPropsType(): SvelteRestProps
 declare function __sveltets_partial<T>(obj: T): Partial<T>;
 declare function __sveltets_partial_with_any<T>(obj: T): Partial<T> & SvelteAllProps
 declare function __sveltets_with_any<T>(obj: T): T & SvelteAllProps
-declare function __sveltets_store_get<T=any>(store: SvelteStore<T>): T
+declare function __sveltets_store_get<T = any>(store: SvelteStore<T>): T
 declare function __sveltets_any(dummy: any): any;
 declare function __sveltets_empty(dummy: any): {};
 declare function __sveltets_componentType(): AConstructorTypeOf<SvelteComponent>
 declare function __sveltets_invalidate<T>(getValue: () => T): T
+declare function __sveltets_eventDef<T extends SvelteEventRecord>(def: T): SvelteOnEvent<T>
```

Comment:
I get a ts error here: `Generic type 'SvelteOnEvent' requires 2 type argument(s)`

```suggestion
declare function __sveltets_eventDef<T extends SvelteEventRecord, K extends keyof T>(def: T): SvelteOnEvent<T, K>
```

Reply:
miss this during rename. Should be `SvelteOnAllEvent<T>`

Reply:
Does this also fix the "cannot use custom events"-bug I described?

Reply:
yeah :grin: